### PR TITLE
Add traps

### PR DESCRIPTION
**Pull request by olliy1or**

### CODE REVIEW

File: mods/ctf/ctf_map/ctf_traps.lua [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	after_place_node = function(pos, placer, itemstack, pointed_thing)
+		local meta = minetest.get_meta(pos)
+		local name = placer:get_player_name()
+
+		meta:set_string("placer", minetest.serialize({
+			team = ctf_teams.get(name),
+			name = name,
+		}))
+	end
+})
```
Comment (Panquesito7) [inline mods/ctf/ctf_map/ctf_traps.lua:90]: Missing an endline.
```suggestion
})

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by RyanZem — APPROVED** [review]

**Review by LoneWolfHT — CHANGES_REQUESTED** [review]
Textures don't follow naming conventions

**Review by LoneWolfHT — CHANGES_REQUESTED** [review]
This should only add the crafts to `classic` and `nade_fight`

**Review by LoneWolfHT — APPROVED** [review]
Just realized the crafts aren't added to any modes by this code, I can do that after this is merged.

File: mods/ctf/ctf_modebase/crafting.lua [hunk 2]
```diff
@@ -167,3 +167,45 @@ for ore, ore_item in pairs(full_ores) do
 		level  = 1,
 	})
 end
+
+-- Traps
```
Comment (savilli) [inline mods/ctf/ctf_modebase/crafting.lua:171]: You don't need `type = "inv"` and `level = 1` anymore

**Pull request merged**